test/e2e/loadtest/client/prom-ingress.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: prometheus
+  namespace: loadtest-e2e
+  labels:
+    application: prometheus
+spec:
+  rules:
+  - host: prometheus-%TARGET%.%ZONE%
```

Comment:
Do we need public host maybe we could use cluster.local?
Then we could also hardcode it in the verification step as it would be the same for any cluster

Reply:
yes we need to query it from the outside (pipeline is executed in another cluster not in e2e)